README.md

```diff
@@ -64,7 +64,7 @@ Using flake, installing `nix-flatpak` as a NixOs module would look something lik
 ```
 
 Depending on how config and inputs are derived `homeManagerModules` import can be flaky. Here's an example of how `homeManagerModules` is imported on my nixos systems config in [modules/home-manager/desktop/nixos/default.nix](https://github.com/gmodena/config/blob/5b3c1ce979881700f9f5ead88f2827f06143512f/modules/home-manager/desktop/nixos/default.nix#L17). `flake-inputs` is a special extra arg set in the repo `flake.nix`
-[mkNixosConfiguration]([https://github.com/gmodena/config/blob/main/flake.nix#L29](https://github.com/gmodena/config/blob/5b3c1ce979881700f9f5ead88f2827f06143512f/flake.nix#L29).
+[mkNixosConfiguration](https://github.com/gmodena/config/blob/5b3c1ce979881700f9f5ead88f2827f06143512f/flake.nix#L29).
```

Comment:
Accept this suggestion if you'd prefer to link to main. Just keep in mind that when `flake.nix` changes, the URL will still point to the same line (i.e. a different line from what it currently does).

```suggestion
[mkNixosConfiguration](https://github.com/gmodena/config/blob/main/flake.nix#L29).
```

Reply:
In this case I'd rather keep pointing to the specific commit id. The purpose is to avoid changes to main (e.g. refactoring my config) would break the example.

FYI: in  the future I plan to avoid linking to my config (it's a bit messy), and add examples to `testing-base`.